schema.graphql

```diff
@@ -22,6 +22,9 @@ type Token @entity {
   # balances: [Balance!]! @derivedFrom(field: "token") // TODO: Pending mapping
   # deposits: [Deposit!]! @derivedFrom(field: "token") // TODO: Disabled for now, until we sort out how to relate it better
 
+  # Cumulative sell volume
```

Comment:
nit: Maybe this can be a doc-comment already (I think they call it a "description" in GraphQL lingo, I'm not sure though)
```suggestion
  "Cumulative sell volume"
```